src/ethereum_test_tools/code/generators.py

```diff
@@ -237,3 +237,99 @@ def __post_init__(self):
         # Finally we append the true and false branches, and the condition, plus the jumpdest at
         # the very end
         self.bytecode = condition_bytes + if_false_bytes + if_true_bytes + Op.JUMPDEST
+
+
+@dataclass
+class BytecodeCase:
```

Comment:
I feel like we could almost get away with calling this `Case`. What do you think?

Reply:
:100: We have a `Switch`, not a `BytecodeSwitch`!